.github/workflows/lint-ratchet.yml

```
@@ -0,0 +1,38 @@
+name: Lint Ratchet
+
+on:
+  pull_request:
+    branches: [main]
+  merge_group:
+
+permissions: read
```

actionlint (1.7.10)

[error] "read" is invalid for permission for all the scopes. available values are "read-all", "write-all" or {}

(permissions)